### PR TITLE
added sl file for list vms

### PR DESCRIPTION
**Pull request by manjoosh12**
Created an SL file for ListVM's operation

### CODE REVIEW

File: content/io/cloudslang/nutanix/prism/virtualmachines/list_vms.sl [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+########################################################################################################################
+#!!
+#! @description: Get a list of Virtual Machines.
+#!
+#! @input protocol: The connection protocol of nutanix.
+#!                  Default: https
```
Comment (patmante) [inline content/io/cloudslang/nutanix/prism/virtualmachines/list_vms.sl:19]: Keep values in '' 
```suggestion
#!                  Default: 'https'
```

Comment (manjoosh12) [reply content/io/cloudslang/nutanix/prism/virtualmachines/list_vms.sl]: done the changes

File: content/io/cloudslang/nutanix/prism/virtualmachines/list_vms.sl [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+########################################################################################################################
+#!!
+#! @description: Get a list of Virtual Machines.
```
Comment (patmante) [inline content/io/cloudslang/nutanix/prism/virtualmachines/list_vms.sl:16]: Add a full description of the operation

Comment (manjoosh12) [reply content/io/cloudslang/nutanix/prism/virtualmachines/list_vms.sl]: added it.

File: content/io/cloudslang/nutanix/prism/virtualmachines/list_vms.sl [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+########################################################################################################################
+#!!
+#! @description: GET Virtual Machine disk information and network information
```
Comment (patmante) [inline content/io/cloudslang/nutanix/prism/virtualmachines/list_vms.sl:16]: Add the following description:

**Get a list of Virtual Machines. Virtual Machine disk information and network information are not included by default as fetching these are expensive operations. These can be included by setting the include_vmdisk_config and include_vmnic_config flags respectively.**

Comment (manjoosh12) [reply content/io/cloudslang/nutanix/prism/virtualmachines/list_vms.sl]: added

**Pull request merged**